src/emulator/controller.ts

```diff
@@ -854,6 +843,16 @@ export async function startAll(
     await startEmulator(loggingEmulator);
   }
 
+  if (showUI && !shouldStart(options, Emulators.UI)) {
+    hubLogger.logLabeled(
+      "WARN",
+      "emulators",
+      "The Emulator UI is not starting, either because none of the emulated " +
+        "products have an interaction layer in Emulator UI or it cannot " +
```

Comment:
Nit: Consider simplifying this message a bit - 'interaction layer' feels like unnecessary jargon here, for example

Reply:
Done